capa/main.py

```diff
@@ -1087,6 +1088,26 @@ def handle_common_args(args):
         args.signatures = sigs_path
 
 
+def last_resort_exception_handler(args, exctype, value, traceback) -> None:   
```

Comment:
an alternative design is to check the args in `main` and install one of two different functions, so you don't have to parse `args` into the handler